oap-server/server-configuration/grpc-configuration-sync/src/main/test/java/org/apache/skywalking/oap/server/configuration/grpc/GRPCConfigurationTest.java

```diff
@@ -0,0 +1,177 @@
+/*
```

Comment:
Please check, the test resource directory structure is wrong, how do you test this locally?

oap-server/server-configuration/grpc-configuration-sync/src/main/test/java/org/apache/skywalking/oap/server/configuration/grpc/GRPCConfigurationTest.java

Reply:
src/main/test is typically wrong

Reply:
:sweat: